cmd/api/main.go

```
@@ -40,9 +44,25 @@ func main() {
+	quit := make(chan os.Signal, 1)
+	signal.Notify(quit, syscall.SIGINT, syscall.SIGTERM)
+	<-quit
+	log.Println("shutdown server ...")
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	if err := server.Shutdown(ctx); err != nil {
+		log.Fatal("server shutdown:", err)
+	}
+	select {
```

golangci-lint

S1000: should use a simple channel send/receive instead of `select` with a single case

(gosimple)